### PR TITLE
Fixes and Test for growthExpMatch

### PR DESCRIPTION
**Pull request by tpfau**
Fixed minor errors in a couple of functions (inconsistence with positions etc).
Also converted the growthExpMatch test to be CI compatible.

**I hereby confirm that I have:**

- [X] Tested my code on my own machine
- [X] Followed the guidelines in the [Contributing Guide](https://opencobra.github.io/cobratoolbox/docs/contributing.html)
- [X] Selected `develop` as a target branch (top left drop-down menu)

*(Note: You may replace [] with [X] to check the box)*

### CODE REVIEW

File: test/verifiedTests/reconstruction/testGrowthExpMatch/testGrowthExpMatch.m [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+%   Adaptions to CI - Thomas Pfau Okt 2017
+
+oriFolder = pwd;
+
+%moves to testing folder that contains testGrowthExpMatch
+test_folder = fileparts(which('testGrowthExpMatch.m'));
+cd(test_folder);
+
+%load Model
+model = readCbModel('ecoli_core_model.mat');
```
Comment (laurentheirendt) [inline test/verifiedTests/reconstruction/testGrowthExpMatch/testGrowthExpMatch.m]: May you use an absolute path here using `CBTDIR`?

File: test/verifiedTests/reconstruction/testGrowthExpMatch/testGrowthExpMatch.m [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+%removes reaction ENO
+disp('------------------------------------')
+disp('Removing reaction, ENO, from test model:')
+model = removeRxns(model, 'ENO');
+
+%moves to folder w/ input files
+% w = what('testing');
+% p = w.path;
+% cd(p);
+d = load('Dictionary.mat');
```
Comment (laurentheirendt) [inline test/verifiedTests/reconstruction/testGrowthExpMatch/testGrowthExpMatch.m]: May you use an absolute path here using `CBTDIR`?

Comment (tpfau) [reply test/verifiedTests/reconstruction/testGrowthExpMatch/testGrowthExpMatch.m]: I'll base it on the test folder, which is definitely stable, while the path relative to CBTDIR can change.

File: test/verifiedTests/reconstruction/testGrowthExpMatch/testGrowthExpMatch.m (hunk 1 above)
Comment (tpfau) [reply test/verifiedTests/reconstruction/testGrowthExpMatch/testGrowthExpMatch.m]: Updated

**Comment by tpfau** [comment]
Changes done

**Pull request merged**